evidence/pandres95/00001-2024-07-retain-rank-1.md

```diff
@@ -0,0 +1,43 @@
+# Evidence-0001: Retention at Rank _I_
+
+|                 |                             |
+| --------------- | ----------------------------|
+| **Report Date** | 2024/07/31                  |
+| **Submitted by**| Pablo Andrés Dorado Suárez  |
+
+## Member details
+
+- Matrix username: `@pandres95:bloque.team`
+- Polkadot address: `12gMhxHw8QjEwLQvnqsmMVY1z5gFa54vND74aMUbhhwN6mJR`
+- Current rank: 1
+- Date of initial induction: [2024/04/19](https://collectives.subsquare.io/fellowship/referenda/105)
+- Date of last report: N/A
+- Area(s) of Expertise/Interest: Assets, NFTs, Governance, XCM
+
+## Reporting period
+
+- Start date: 2024/04/19
+- End date: 2024/07/31
+
+## Evidence
+
+| Areas of Contribution | Tasks | Links | Notes |
+| --------------------- | ----- | ----- | ----- |
+| Fellowship Runtimes   | Allow Sending XCM messages using a Signed origin on Kusama | [Runtimes #290](https://github.com/polkadot-fellows/runtimes/pull/290) | |
+| Polkadot SDK          | [Assets] Call implementation for `transfer_all` | [Polkadot SDK #4527](https://github.com/paritytech/polkadot-sdk/pull/4527) | _Approved_ / **In Auditing** ? |
+| Polkadot SDK          | [Assets] Implement `pallet-assets-holder` | [Polkadot SDK #4530](https://github.com/paritytech/polkadot-sdk/pull/4530) | _In Progress_ / **Waiting for Review** |
+| Fellowship Runtimes   | Implement Unbrick Collective | [Runtimes #376](https://github.com/polkadot-fellows/runtimes/issue/376) | _Preparing_ / **Gathering requirements** |
+
+## Voting record
+
+N/A
+
+## Misc
+
+- [ ] Question(s):
+
+- [x] Concern(s):
+  - Code Reviews / Merge Trains can be sometimes a bit slow, this might potentially affect what can be marked as completed by the time the evidence report comes.
+
+- [x] Comment(s):
+    Been as actively in the monthly calls as I can humanly be (including the last in-person meeting).
```

Comment:
Please use [objective language](https://www.grammarly.com/blog/objective-vs-subjective/).

```suggestion
Consistent active participation to monthly Fellowship calls.
```
or even (if that's the case)
```suggestion
Perfect attendance to monthly Fellowship calls with active participation or input to most/all of them.
```